assets/sass/layout.scss

```diff
@@ -317,6 +317,10 @@ p.center {
   }
 }
 
+svg.graphviz {
+  width: 100%;
+}
+
```

Comment:
is the reason for this that the SVG width gets a bit weird because graphviz hardcodes the svg width? I'm not sure about this since having a width of 100% won't always be the right approach in every context.

One alternative could be base64-encoding the images and using `<img src="<base64-encoded svg">`. What I like about the `<img>` option is that I find that `<img>` tags are usually easier to position with CSS than `<svg>` tags. Also then we could reuse the existing dark mode filters for `<img>` tags.

Reply:
It also makes it easy to "open image in tab". I just tried it, and wrapping the existing `<svg>` block in `<img src='data:image/svg+xml;utf8,<svg>...</svg>' />` works (because the `<svg>` only contains double quotes, I can wrap it in a single-quoted attribute)..

Reply:
> I noticed though that saving such svg to file gives you `Untitled.svg` (on Firefox). But there doesn't seem to be a workaround for that at the moment: [whatwg/html#2722](https://github.com/whatwg/html/issues/2722)

Right, that's a bummer.

> Inlining SVG has the advantage of being able to style the elements with CSS. So this could make it a lot easier to style better dark mode colors.

Indeed, I tried to make this work by following https://getpublii.com/docs/prepare-svg-for-light-dark-mode.html, but failed, probably because I did not _really_ change the _system_ to dark mode, but simply clicked the dark mode button... And as you say, an inlined `<svg>` can share the style with the HTML, but an SVG inside an `<img>` cannot.